USER_GUIDE.md

```diff
@@ -195,3 +197,44 @@ cloudai --mode generate-report \
 ```
 
 `--output-dir` accepts one scenario run results directory.
+
+## Installing the NeMo Datasets (Pile or mC4)
+Basically, you can install the NeMo launcher test template by following the README file of CloudAI. However, in addition to the steps, you need to download NeMo datasets manually. To understand datasets available in the NeMo framework, you can take a look at the Data Preparation section of [the document](https://docs.nvidia.com/nemo-framework/user-guide/latest/llms/baichuan2/dataprep.html). According to the document, NeMo supports two types of datasets depending on the model that you want to train: The Pile and mC4. It also provides detailed instructions on how to download these datasets for various platforms. Let’s assume that we have a slurm cluster.
+
+You should have at least 1TB space in your data directory (data_dir) as the Pile dataset size is 800GB. Alternatively, you can control the file numbers to download: link
+
+Please make sure to clean up your data_dir clean before submitting the job:
+```bash
+$ rm -rf /mswg2/E2E/theo/*
+```
+
+You can download the datasets with the following command:
+```bash
+$ python3 launcher_scripts/main.py \
```

Comment:
Should we suggest using the same python we use for Cloud AI CLI? And that suggests, that we need to add a section that describes how to initialise it like we have in README.

Reply:
CloudAI CLI cannot download the datasets. It is not CloudAI's responsibility. Please review the updated USER_GUIDE.md.